src/operator/tensor/elemwise_binary_op-inl.h

```diff
@@ -374,6 +374,72 @@ void ElemwiseBinaryOp::CsrCsrOp(mshadow::Stream<cpu> *s,
   }
 }
 
+template<typename OP>
+struct ElemwiseDnsMapKernel {
```

Comment:
I do not have a better name for this kernel, this kernel is computing the output value for the dense value with 0. Any suggestions?